ansible/roles/flannel/handlers/main.yml

```diff
@@ -14,8 +35,21 @@
   command: ip link delete docker0
   when: '"docker0" in ansible_interfaces'
 
-- name: start docker
+  # TODO: work only on default installation...
+- name: docker graph driver 
+  command: findmnt -no FSTYPE -T /var/lib/docker
```

Comment:
Since flannel set-up invokes docker restart, I assume fix for docker goes here respectively.
There is possibility that docker installed before docker role (manually/by other play etc).
Solution for code de-duplication You can see in my fork on ansible/rewrite